web/src/hooks/data/useFetchP2PPaymethods.tsx

```diff
@@ -0,0 +1,19 @@
+import { p2pUrl } from 'web/src/api/config';
+import { fetchWithCreds } from 'web/src/helpers/fetch';
+import { AdvertType, PaymethodSource } from 'web/src/modules/p2p/types';
+import { useFetch } from './useFetch';
+
+interface P2PPaymethodsParams {
+  type: AdvertType | null;
+  currency: string | null;
+  cryptocurrency: string | null;
```

Comment:
На данном уровне лучше `undefined` использовать

```suggestion
  type?: AdvertType | undefined;
  currency?: string | undefined;
  cryptocurrency?: string | undefined;
```